src/client/ui/styles.less

```diff
@@ -659,6 +670,55 @@
                 }
             }
         }
+
+        .selector-panel-toggle-button {
+            background: white;
+            position: absolute;
+            height: 21px;
+            width: 70px;
```

Comment:
Using width here is not needed. Everythink should work once we remove `position absolute` from nested span